### PR TITLE
Profilecont2

### PR DESCRIPTION
**Pull request by carlbenjaminlyon**
Working: 
1. Google Login returns unique profile id, can likely get google supplied object instead
2. Profile.tsx is getting user favorite data from related tables from dummy user object id

Todo:
1. Need to create route for updating a users favorites for each category
2. Display user favorited items in persistent view, where user can remove them

If need be, I can display this part locally, and we can merge it after tomorrows presentations if it looks good.

### CODE REVIEW

**Review by sbelete01 — COMMENTED** [review]
Why did you change the findAll's to findOne's?

**Comment by carlbenjaminlyon** [comment]
> Why did you change the findAll's to findOne's?

When searching a route by the given parameters, kept getting all the users present in the DB, along with their respective actors, instead of a single user with an actors array. There was a help desk ticket about it with Peter.

File: server/index.ts [hunk 1]
```diff
@@ -9,13 +9,22 @@ import MoviesRouter from './api/movies';
...
 import DirectorsRouter from './api/directors';
 import GenresRouter from './api/genres';
+import passport from 'passport';
+import passportLocal from 'passport-local';
+import cookieParser from 'cookie-parser';
+import session from 'express-session';
+import formData from 'express-form-data';
+// const dotenv = require('dotenv');
+const app = express();
+import { Response, Request } from 'express';
```
Comment (badams92) [inline server/index.ts]: this should be `import type`

**Comment by badams92** [comment]
Looks good. We'll have to confirm `findOne` vs `findAll` with more testing

File: server/index.ts (hunk 1 above)
Comment (carlbenjaminlyon) [reply server/index.ts]: Just updated this, now reading
`import type { Response, Request} from 'express';`

**Comment by badams92** [comment]
like the new changes

**Pull request closed without merging**